opencti-platform/opencti-front/src/components/CKEditor.tsx

```diff
@@ -0,0 +1,217 @@
+import { CKEditor as ReactCKEditor } from '@ckeditor/ckeditor5-react';
+import {
+  Editor,
+  Alignment,
+  Autoformat,
+  AutoImage,
+  AutoLink,
+  Base64UploadAdapter,
+  BlockQuote,
+  Bold,
+  ClassicEditor,
+  Code,
+  CodeBlock,
+  Essentials,
+  FontBackgroundColor,
+  FontColor,
+  FontFamily,
+  FontSize,
+  Heading,
+  Highlight,
+  HorizontalLine,
+  ImageCaption,
+  ImageInsert,
+  ImageResize,
+  ImageStyle,
+  ImageToolbar,
+  ImageUpload,
+  Indent,
+  IndentBlock,
+  Italic,
+  Link,
+  LinkImage,
+  ListProperties,
+  Mention,
+  Paragraph,
+  PasteFromOffice,
+  RemoveFormat,
+  SourceEditing,
+  SpecialCharacters,
+  SpecialCharactersCurrency,
+  SpecialCharactersEssentials,
+  Strikethrough,
+  Subscript,
+  Superscript,
+  List,
+  Table,
+  TodoList,
+  TableCaption,
+  TableColumnResize,
+  TableToolbar,
+  Underline,
+  ImageEditing,
+  ImageBlockEditing,
+} from 'ckeditor5';
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+// eslint-disable-next-line import/extensions
+import de from 'ckeditor5/translations/de.js';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+// eslint-disable-next-line import/extensions
+import en from 'ckeditor5/translations/en.js';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+// eslint-disable-next-line import/extensions
+import es from 'ckeditor5/translations/es.js';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+// eslint-disable-next-line import/extensions
+import fr from 'ckeditor5/translations/fr.js';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+// eslint-disable-next-line import/extensions
+import ja from 'ckeditor5/translations/ja.js';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+// eslint-disable-next-line import/extensions
+import ko from 'ckeditor5/translations/ko.js';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+// eslint-disable-next-line import/extensions
+import zh from 'ckeditor5/translations/zh.js';
```

Comment:
Yes, that's disgusting, I know 🤢 but cannot found a better way with how ckeditor is bundled...